src/sth_database.js

```diff
@@ -326,6 +327,9 @@
     var connection = mongoose.connection.useDb(databaseName);
 
     // Get the connection and notify it via the callback.
+    sthLogger.debug('Getting access to the collection: ' + collectionName, {
```

Comment:
Is this log related to @mcarracedo requirement? I can remember him asking for a log printing the accessed collection and database. If it is the case, I think the database name is missing in the log :)